src/Surfnet/StepupGateway/GatewayBundle/Saml/Proxy/ProxyStateHandler.php

```diff
@@ -27,7 +27,7 @@ class ProxyStateHandler
     /**
      * @var \Symfony\Component\HttpFoundation\Session\SessionInterface
      */
-    private $session;
+    protected $session;
```

Comment:
Is there a specific reason for exposing this property to subclasses at this point?